### PR TITLE
i18n(es): Updated `netlify.mdx` & `partytown.mdx`

### PR DESCRIPTION
**Pull request by dreyfus92**
#### What kind of changes does this PR include?

- New or updated content

#### Description

- What does this PR change? Give us a brief description.
Updated `netlify.mdx` & `partytown.mdx` based on this commit b0cd40b14e9a850d23b14cb9a0ab01a0db2de2cb

### CODE REVIEW

File: src/content/docs/es/guides/integrations-guide/partytown.mdx [hunk 1]
```diff
@@ -131,7 +131,7 @@ export default defineConfig ({
 
 ## Ejemplos
 
-*   [Explora proyectos con Astro Partytown en GitHub](https://github.com/search?q=%22%40astrojs%2Fpartytown%22+path%3A**%2Fpackage.json&type=code) para ver más ejemplos.
+*   ¡[Explora proyectos con Astro Partytown en GitHub](https://github.com/search?q=%22%40astrojs%2Fpartytown%22+path%3A**%2Fpackage.json&type=code) para ver más ejemplos!
```
Comment (Waxer59) [inline src/content/docs/es/guides/integrations-guide/partytown.mdx:134]:
```suggestion
*   ¡[Explora proyectos con Astro Partytown en GitHub](https://github.com/search?q=%22%40astrojs%2Fpartytown%22+path%3A**%2Fpackage.json\&type=code) para ver más ejemplos!
```

**Review by Waxer59 — COMMENTED** [review]
Buena PR 🤠, te dejo unas sugerencias

**Review by Waxer59 — APPROVED** [review]
LGTM ✔

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Thanks peeps 🚀

**Pull request merged**